samza-sql/src/main/java/org/apache/samza/sql/interfaces/UdfMetadata.java

```diff
@@ -32,6 +32,7 @@
 public class UdfMetadata {
 
   private final String name;
```

Comment:
Might be better to rename the field `name ` to something more meaningful(like udfLogicalName or something else).

Reply:
Added a description of what this field means. Well, it is just udf name. For UdfMetadata, it should not matter whether it is logicalName or a physicalName, it is just the name that the udf will be referred as. I feel name is appropriate. What do you think ?